General/Enum.stencil

```diff
@@ -32,35 +32,36 @@ extension RawStringConvertible  {
                 reason: "Expected a String"
             )
         }
-        
+
         guard let state = Self.init(rawValue: string) else {
             throw Abort(
                 .internalServerError,
                 metadata: nil,
                 reason: "Invalid case: \(string)"
             )
         }
-        
+
         self = state
     }
-    
+
     public func makeNode(in context: Context?) throws -> Node {
         return .string(self.rawValue)
     }
 }
 
 // MARK: Enums
-
 {% for enum in types.enums|!annotated:"ignore" where arguments.enumOptOut or enum.annotations.enum or enum.based.RawStringConvertible %}
 
 extension {{ enum.name }} {
     {{ type.accessLevel }} static var all: [{{ enum.name }}] = [
         {% for case in enum.cases %}
-        .{{ case.name }},
+        .{{ case.name }}{% if not forloop.last %},{% endif %}
         {% endfor %}
     ]
+    {% if enum.rawTypeName != nil %}
 
```

Comment:
Should we remove this newline?

Reply:
You mean the one between the `]` and `static let allRaw`? It seems better to me like this. I do want to remove the newline _after_ each enum extension though... coming up!

Reply:
Ah sorry, I wasn't paying enough attention. I totally agree 👍